### ISSUE TITLE
[BUG] Charges are not spent

### ISSUE THREAD

**Issue by LupusVentus** [PwQt/magic-items-2#151]
**Module Version:** v4.1.5.1
**System Version:** DnD 3.2.0
**Foundry Version:** 11.315
Midi 11.4.32

**Before open any issue**
No distinct message appears in the console

**Describe the bug**
No charges of a magic item are spent if using a single target action, while having a targeted token beforehand.

**To Reproduce**
1. Create a magic item
2. Add a single target action that uses charges (i.e. ranged attack)
3. Target any creature and use the action
4. See how action gets processed, but no charges get spent

**Comment by PwQt** [comment]
@LupusVentus do you mean charges you've set up in Item or in "Magic Item" tab in the item?

**Comment by nschoenwald** [comment]
> @LupusVentus do you mean charges you've set up in Item or in "Magic Item" tab in the item?

Not the original creator of the issue, but I think the expected behavior should be that charges on the item are synced with the "Magic item" charges or that the module just uses the charges on the item, there's no reason for two different values from a user perspective.

**Comment by nschoenwald** [comment]
If I understand correctly, this is the same issue: https://github.com/PwQt/magic-items-2/issues/131

**Comment by PwQt** [comment]
Yes, I'm planning as a next major "upgrade" to do an interim period of having a choice of "internal" charges with the current system, and it'll gradually be phased out of.

**Comment by PwQt** [comment]
Charges on item are now supported - https://github.com/PwQt/magic-items-2/releases/tag/4.3.0
